apps/rush/src/actions/RebuildAction.ts

```diff
@@ -148,17 +148,32 @@ export default class RebuildAction extends CommandLineAction {
       () => {
         stopwatch.stop();
         console.log(colors.green(`rush ${this.options.actionVerb} (${stopwatch.toString()})`));
+        this._collectTelemetry(stopwatch, true);
       },
       () => {
         stopwatch.stop();
         console.log(colors.red(`rush ${this.options.actionVerb} - Errors! (${stopwatch.toString()})`));
-        process.exit(1);
+        this._collectTelemetry(stopwatch, false);
+        this._parser.exitWithError();
       })
       .then(() => {
         this._eventHooksManager.handle(Event.postRushBuild);
       });
   }
 
+  private _collectTelemetry(stopwatch: Stopwatch, success: boolean): void {
+    this._parser.telemetry.log({
```

Comment:
>this._parser.telemetry.log [](start = 4, length = 26)

in evergreen I ended up making a base class for Actions, rather than sticking everything on _parser. It seemed cleaner